File: Almacen/Almacen/Form1.vb
Imports System.Data.SQLite

Public Class Form1

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Dim sqlconsulta As String
        sqlconsulta = "SELECT * FROM usuarios where login = '" + TextBox1.Text + "' and password = '" + TextBox2.Text + "'"


        Dim connection As String = "Data Source=Almacen.db;Version=3;New=False;Compress=True;"

        Dim SQLConn As New SQLiteConnection(connection)
        Dim SQLcmd As New SQLiteCommand(SQLConn)
        SQLcmd.CommandText = sqlconsulta
        'Dim SQLdr As SQLiteDataReader
        If SQLConn.State <> ConnectionState.Open Then
            SQLConn.Open()
        End If
        Dim dt As New DataTable()
        Dim ds As New DataSet()
        Dim da As New SQLiteDataAdapter(SQLcmd)
        da.Fill(dt)
        If dt.Rows.Count <= 0 Then
            MessageBox.Show("Usted ha ingresado un Usuario o una Contraseña erroneos,Vuelva a intentarlo")
            TextBox1.Text = ""
            TextBox2.Text = ""
        Else
            Dim campos As String() = New String(dt.Rows.Count - 1) {}
            For i As Integer = 0 To dt.Rows.Count - 1

                If DirectCast(dt.Rows(i)("login"), [String]).Equals(TextBox1.Text) And DirectCast(dt.Rows(i)("password"), [String]).Equals(TextBox2.Text) Then
                    Form2.Show()
                    Me.Finalize()
                End If
            Next
            SQLConn.Close()
        End If
    End Sub

    Private Sub TextBox2_KeyDown(sender As Object, e As KeyEventArgs) Handles TextBox2.KeyDown
        If e.KeyValue = 13 Then
            Button1_Click(Nothing, Nothing)
        End If

    End Sub
End Class

File: Inventario/Inventario/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Inventario
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Button1_Click(System.Object sender, System.EventArgs e)
        {
            string usuario = null;
            string contrasena = null;
            string usu = null;
            string contra = null;
            string usua = null;
            string contrase = null;

            usu = TextBox1.Text;
            contra = TextBox2.Text;
            usuario = "administrador";
            contrasena = "admin";
            usua = "usuario";
            contrase = "usuario";


            if (usu == usuario & contra == contrasena)
            {
                MessageBox.Show("Usted es Administrador");
                Form2.Show();
                this.Finalize();
            }
            else
            {
                if (usu == usua & contra == contrase)
                {
                    MessageBox.Show("Usted es Usuario");
                    Form2.Show();
                    Form2.MercanciaToolStripMenuItem.Enabled = false;
                    //Form3.Button4.Enabled = False
                    //Form3.Button2.Enabled = False
                    //Form5.Button4.Enabled = False
                    this.Finalize();
                }
                else
                {
                    if (usu != usuario | usu != usua | contra != contrasena | contra != contrase)
                    {
                        MessageBox.Show("Usted ha ingresado un Usuario o una Contraseña erroneos");
                        MessageBox.Show("Vuelva a intentarlo");
                    }
                }

            }
        }
    }
}
